<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>225</height>
      </size>
     </property>
     <property name="title">
      <string>Tetszőleges koordináta rendszerek hozzáadása</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="addableLinesListWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QToolButton" name="addLineToolButton">
          <property name="toolTip">
           <string>Kiválasztott egyenesek hozzáadása</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="arrowType">
           <enum>Qt::RightArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeLineToolButton">
          <property name="toolTip">
           <string>Kiválasztott egyenesek törlése</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="addedLinesListWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QToolButton" name="addCoordinateSystemToolButton">
          <property name="toolTip">
           <string>Kiválasztott egyenesekből koordinátarendszer létrehozása</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="arrowType">
           <enum>Qt::RightArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeCoordinateSystemToolButton">
          <property name="toolTip">
           <string>Kiválasztott koordinátarendszer törlése</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="coordinateSystemsTreeWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" rowspan="3">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Modellek</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="ASADCheckBox">
          <property name="text">
           <string>AS - AD</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ISLMCheckBox">
          <property name="text">
           <string>IS - LM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="SICheckBox">
          <property name="text">
           <string>Nemeti megtakarítás, 
beruházás</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" rowspan="3">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Piacok</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="labourCheckBox">
          <property name="text">
           <string>Munkapiac</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_15">
          <property name="text">
           <string>Árupiac</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_16">
          <property name="text">
           <string>Pénzpiac</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ISRCheckBox">
          <property name="text">
           <string>Beruházási kereslet, 
kínálat</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Egyenletek</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="outputCheckBox">
          <property name="text">
           <string>Termelési függvény</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="ADCheckBox">
          <property name="text">
           <string>AD görbe</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="phillipsCheckBox">
          <property name="text">
           <string>Phillips - görbe</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="keynesCheckBox">
          <property name="text">
           <string>Keynesi kereszt</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="SRASCheckBox">
          <property name="text">
           <string>SRAS görbe</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="ISCheckBox">
          <property name="text">
           <string>IS görbe</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="LRASCheckBox">
          <property name="text">
           <string>LRAS görbe</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="LMCheckBox">
          <property name="text">
           <string>LM görbe</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addLineToolButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>addLines()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeLineToolButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>removeLines()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addCoordinateSystemToolButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>addCoordinateSystem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeCoordinateSystemToolButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>removeCoordinateSystem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addLines()</slot>
  <slot>removeLines()</slot>
  <slot>removeCoordinateSystem()</slot>
  <slot>addCoordinateSystem()</slot>
 </slots>
</ui>
